portfolio/pom.xml

```diff
@@ -38,8 +38,14 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
-  </dependencies>
+  
 
+<dependency>
+  <groupId>com.google.code.gson</groupId>
+  <artifactId>gson</artifactId>
+  <version>2.8.6</version>
+</dependency>
```

Comment:
Nit: should be indented. As a good rule of thumb, always run auto-format before making a PR -- helps prevents these types of comments